### ISSUE TITLE
Nipun Batra

### ISSUE THREAD

**Issue by guevara** [guevara/read-it-later#4102]
Nipun Batra<br>
<br>
http://bit.ly/2M3XPg2<br>
<br>
About 0 Minutes<br>
<br>
<div>
<div>
<div>
<div>
<p>We all get it - AI is the new electricity. Deep neural nets are everywhere around us. But you know what, getting labelled training data can still be a big issue in many domains. This is where active learning comes in - given that we only have a small amount of labelled data, do we randomly get labels for other samples, or can we create a <code>smarter</code> strategy for the same? Active learning deals with the latter.</p>
<p>Various strategies for active learning have been proposed in the past. In this post, I'll work out a trivial example of what is called query by committee. The key idea is that we create a committee of learners and choose to acquire labels for the unlabelled points for which there is maximum disaggrement amongst the committee.</p>
<p>I'd recommend the new readers to go through <a href="http://burrsettles.com/pub/settles.activelearning.pdf">this</a> survey.</p>
<p>In this particular post, I'd be looking at active learning via query by committee, where the committee members are trained on different subsets of the train data. In a future post, I'll write about active learning via query by committee, where the committee members are trained on the same data, but with different parameters.</p>
</div>
</div>
</div>
<div>
<div>
<div>
<h4>Learning a linear regression model on the entire data<a href="#Learning-a-linear-regression-model-on-the-entire-data">&para;</a>
</h4>
</div>
</div>
</div>
<div>
<div>
<div>
<h4>Creating the initial train set, the test set and the pool<a href="#Creating-the-initial-train-set,-the-test-set-and-the-pool">&para;</a>
</h4>
</div>
</div>
</div>
<div>
<div>
<div>
<h4>Creating a committee each learnt on different subset of the data<a href="#Creating-a-committee-each-learnt-on-different-subset-of-the-data">&para;</a>
</h4>
</div>
</div>
</div>
<div>
<div>
<div>
<h4>Plotting the fit of the committee on the entire dataset<a href="#Plotting-the-fit-of-the-committee-on-the-entire-dataset">&para;</a>
</h4>
</div>
</div>
</div>
<div>
<div>
<div>
<h4>Evaluate the performance on the test set<a href="#Evaluate-the-performance-on-the-test-set">&para;</a>
</h4>
</div>
</div>
</div>
<div>
<div>
<div>
<h4>Active learning procedure<a href="#Active-learning-procedure">&para;</a>
</h4>
</div>
</div>
</div>
</div>
<br>
<br>
<br>
<br>
<br>
<br>
via nipunbatra.github.io http://bit.ly/2Eq3lnf<br>
<br>
May 23, 2019 at 02:56PM
